l10n_nl_postcodeapi/tests/test_l10n_nl_postcodeapi.py

```diff
@@ -0,0 +1,202 @@
+# Copyright 2018-2020 Onestein (<https://www.onestein.eu>)
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+from types import SimpleNamespace
+from unittest.mock import patch
+
+from odoo.modules.module import get_module_resource
```

Comment:
According to https://github.com/odoo/odoo/blob/13.0/odoo/modules/module.py#L219-L220, I think it's better to use `get_resource_path` instead of `get_module_resource`.